### DISCUSSION TITLE
Aqara/Xiaomi announced it will push Thread/Matter support to existing Zigbee-based via OTA firmware updates

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#925] [General]
FYI, Aqara/Xiaomi announced it will push Thread/Matter OTA (Over-The-Air) firmware updates to some of its existing Zigbee-based products, and they also plan on releasing a new line-up of Thread/Matter based sensor devices sometime in the second half of 2022. 

Source: https://www.aqara.com/us/news/article/1493841389451329536

Thread/Matter OTA updates will be pushed via new Aqara existing M2/M1S hubs and gateways (and future Aqara Zigbee hubs/gateways). 

"_Aqara has always offered a series of innovative, home automation products that work with major third-party ecosystems including Apple HomeKit, Amazon Alexa and Google Home. As the industry continues to evolve toward a unified IoT standard, Aqara will continue to support smart home interoperability. In September 2021, Aqara has announced its commitment to Matter, a new industry standard for smart home devices that is developed and led by some of the biggest technology companies._

***As part of its commitment, Aqara will bring Matter support to its existing Zigbee-based products by pushing the over-the-air (OTA) update to the Aqara hubs, the M2 and the M1S`*`. The software update will expose the Aqara child devices to Matter, allowing them to connect with other Matter-compatible third-party devices. This approach assures a seamless interaction between the two protocols, and Aqara users will be able to keep their existing Zigbee devices while accessing the benefits of Matter.***

_The new Thread-based product lineup from Aqara provides users one more option to enter the Matter ecosystem. As one of the pillar connectivity technologies for Matter, Thread enables IoT devices to connect natively to Matter. The Aqara Thread products also integrate Bluetooth Low Energy for a better connection and operating experience._

_Moreover, the Aqara Thread devices will still have some of the same and most popular features seen from the Zigbee product line, such as the long battery life and low-latency connection, while also bringing reliability to a new level with Thread's mesh networking and "self-heal" ability._

_As a partner of NXP Semiconductors, Aqara will showcase some of its latest Thread-based sensors at the NXP demo booth during CES 2022._

**_`*` Certain Aqara products may not be compatible with Matter due to the lack of Matter support for specific device type(s)._**"

PS: This news was first spotted at https://smarthomescene.com/news/aqara-announces-thread-support-in-new-product-line-up/

**Comment by MattWestb** [comment]
If i have not misunderstanding is one OTA update of the **GW** for getting most existing devices tunneled to Thread/Matter and can being used with other networks. Its no word of updating existing Zigbee devices at all.
So its little like the Silabs approach that is under test in HA tunneling Zigbee to bellows and using it as normal Zigbee devices but is also open for native Thread/Matter devices.  I have not testing the Silabs version (more then compiling one firmware that i have not testing for My Billys) if its also supporting native Thread/Matter devices and connecting them to bellows.

**Comment by kevinfishhhhh** [comment]
Aqara and Xiaomi is not the same company
